javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,131 @@
+let info = [{       // User name array
+    name: ""
+}]
+
+let todolist = []       // Todo list array
+
+function getReply(command) {
+    let arr = []        // Users inputs arrays
+    command1 = command.split(' ')
+    arr.push(command1)
+
+    switch (arr[0][0] + arr[0][1] + arr[0][2] + arr[0][3]) {
+
+        // Introduce inputs
+
+        case 'Hellomynameis':
+            if (info[0].name == arr[0][4]) {
+                console.log(`Okey okey i got it. Your name is ${info[0].name}. You dont need to repeat that.`)
+            }
+            else {
+                info[0].name = arr[0][4]
+                console.log(`Nice to meet you ${info[0].name}`)
+            }
+            break
+
+        // User name
+
+        case 'Whatismyname':
+            if (info[0].name == "") {
+                console.log('How can i know that. You dont introduce yourself!!!')
+            }
+            else {
+                console.log(`Your name is ${info[0].name}`)
+            }
+            break
+
+        // Set Timer
+
+        case 'Setatimerfor':
+            if (arr[0][5] == 'minutes') {
+                time = arr[0][4] * 60000
+            }
+            else if (arr[0][5] == 'seconds') {
+                time = arr[0][4] * 1000
+            }
```

Comment:
If you wanna make these cases more readable, you could assign the value before using it:
```js
case 'setatimerfor':
  const timeNumber = commandWords[4];
  const timeWord = commandWords[5];
  if (timeWord == 'minutes') {
    time = timeNumber * 60000
  } else if (timeWord == 'seconds') {                
    time = timeNumber * 1000            
  }
```